src/main/java/org/galatea/starter/service/IexClient.java

```diff
@@ -30,6 +32,22 @@
    * @return a list of the last traded price for each of the symbols passed in.
    */
   @GetMapping("/tops/last")
-  List<IexLastTradedPrice> getLastTradedPriceForSymbols(@RequestParam("symbols") String[] symbols);
+  List<IexLastTradedPrice> getLastTradedPriceForSymbols(@RequestParam("symbols") String[] symbols,
```

Comment:
I don't think we need to provide the token for this endpoint. Do we?

Reply:
These endpoints don't require a token when using api.iextrading.com, but they do require a token when using cloud.iexapis.com. I wrote the historicalPrices endpoint based on the cloud.iexapis.com documentation and decided to update getAllSymbols and getLastTradedPrice to use cloud.iexapis.com as well.

Reply:
Gotcha! I also like that you had it in a separate commit. Very clear